mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -162,6 +164,27 @@ class BackgroundTasksManager : BroadcastReceiver() {
                     resultCode = TaskerPlugin.Setting.RESULT_CODE_PENDING
                 }
             }
+            ACTION_VOICE_RESULT -> {
+                var voiceCommand = intent.getStringArrayListExtra(RecognizerIntent.EXTRA_RESULTS)?.elementAtOrNull(0)
+                    ?: return
+
+                Log.i(TAG, "Recognized text: $voiceCommand")
+                context.getPrefs().getPrefixForVoice()?.let { prefix ->
+                    voiceCommand = "$prefix|$voiceCommand"
+                    Log.d(TAG, "Prefix voice command: $voiceCommand")
+                }
```

Comment:
Since voice commands are handled specially anyway, why not remove adding the prefix here...

Reply:
Sure, that's a better solution.